charts/langsmith/templates/backend/auth-bootstrap.yaml

```diff
@@ -0,0 +1,96 @@
+{{- if .Values.backend.authBootstrap.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "langsmith.fullname" . }}-{{ .Values.backend.name }}-basic-auth-bootstrap
+  labels:
+    {{- include "langsmith.labels" . | nindent 4 }}
+    {{- with.Values.backend.authBootstrap.labels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "0"
+    {{- include "langsmith.annotations" . | nindent 4 }}
+    {{- with.Values.backend.authBootstrap.annotations }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  backoffLimit: 3
+  template:
+    metadata:
+      {{- with .Values.backend.authBootstrap.annotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- with.Values.backend.authBootstrap.labels }}
+            {{- toYaml . | nindent 8 }}
+        {{- end }}
+        {{- include "langsmith.labels" . | nindent 8 }}
+        app.kubernetes.io/component: {{ include "langsmith.fullname" . }}-{{ .Values.backend.name }}
+    spec:
+      restartPolicy: Never
+      {{- with .Values.images.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.backend.authBootstrap.podSecurityContext | nindent 8 }}
+      serviceAccountName: {{ include "backend.serviceAccountName" . }}
+      containers:
+        - name: auth-bootstrap
+          {{- with .Values.backend.authBootstrap.command }}
+          command:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          env:
+            {{- include "langsmith.commonEnv" . | nindent 12 }}
+            - name: BASIC_AUTH_ENABLED
```

Comment:
should this be in our commonEnv or envConfigmap(they go in configmap if we want to override in our saas app)

Reply:
I think commonEnv? won't it be by default `false` for the SaaS version based on `values.yaml`, so no need to add to configmap?

Reply:
agreed lets put in commonEnv then(does seem like its used in multiple places)